docs/api/session.md

```diff
@@ -348,8 +348,14 @@ win.webContents.session.setCertificateVerifyProc((request, callback) => {
 
 * `handler` Function | null
   * `webContents` [WebContents](web-contents.md) - WebContents requesting the permission.  Please note that if the request comes from a subframe you should use `requestingUrl` to check the request origin.
-  * `permission` String - Enum of 'media', 'geolocation', 'notifications', 'midiSysex',
-    'pointerLock', 'fullscreen', 'openExternal'.
+  * `permission` String - Types of permission that can be requested
```

Comment:
can you add a period here for consistency?

Reply:
this should also be formatted like so:
```
`permission` String - Types of permission that can be requested.
   * `media` - Allows app to access media files from a user's device with the user's consent.
   * etc
```

There's no need to redundantly specify string

```